0000-template.md

```diff
@@ -1,102 +1,118 @@
 ---
-feature name: (fill me in with a unique identifier, my-awesome-feature)
-start date: (fill me in with today's date, YYYY-MM-DD)
-rfc pr: (leave this empty)
-related issue: (tracking issue number)
+rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/xxx <-- fill this after you've already created the PR
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/yyy
 ---
 
-# Summary
+# [TITLE]
 
-> Brief description of the feature.
+> One sentence: brief description of the feature from a user perspective.
 
-# README
+## Working Backwards
 
-> Work backwards from the README: pretend that this feature is already
-> implemented and write the README section. It will help you think about your
-> proposal from a user experience point of view. If a README format does not
-> make sense for your proposal, write a "Press Release" which announces your new
-> feature instead.
+> This section should contain one or more "artifacts from the future", as if the
+> feature was already released and we are publishing its CHANGELOG, README
+> and optionally a PRESS RELEASE.
+>
+> This is the most important section of your RFC. It's a powerful thought
+> exercise which will challenge you to truly think about this feature from a
+> user's point of view.
 
-# Motivation
+### CHANGELOG
 
-> Why are we doing this? What use cases does it support? What is the expected
-> outcome?
+> Write the changelog entry for this feature in conventional form (e.g.
+> `feat(eks): cluster tags`).
+>
+> If this change includes a breaking change, include a `BREAKING CHANGE` clause
+> with information on how to migrate. If migration is complicated, refer to a
+> fictional GitHub issue and add its contents here.
 
-# Design Summary
+### README
```

Comment:
Do we need the `README` section in addition to the content of the `PRESS RELEASE` and `Working Backwards`,  I feel like there will be a lot of duplicate between the two.

Reply:
See line 12, the requirement is to include "one or more artifacts", not all of them.